packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js

```diff
@@ -475,7 +469,7 @@ class DemoWidget extends Component {
                   inputSize="small-12"
                   type="number"
                   onChange={this.handleRecentSpaceLoadCountChange}
-                  value={this.state.spaceLoadCount}
+                  placeholder={this.state.spaceLoadCount}
```

Comment:
What was the benefit of changing this to `placeholder` from `value`?

Reply:
addressed in the space.